Content/default/tests/Client/Client.Tests.fs

```diff
@@ -5,27 +5,27 @@ open Fable.Mocha
 open Index
 open Shared
 
-let client = testList "Client" [
-    testCase "Added todo" <| fun _ ->
-        let newTodo = Todo.create "new todo"
-        let model, _ = init ()
+let client =
+    testList "Client" [
+        testCase "Added todo" (fun _ ->
```

Comment:
I changed from `<|` to `()` in these cases otherwise Fantomas produced something pretty wacky.